modular_skyrat/modules/reagent_forging/code/forge_items.dm

```diff
@@ -1,3 +1,26 @@
+GLOBAL_LIST_INIT(allowed_forging_materials, list(
```

Comment:
should this be alphabetized or otherwise sorted?

Reply:
It is sorted just in "oh right I need to add that material too" order